api/install/v1alpha1/eventingester_webhook.go

```diff
@@ -44,10 +46,36 @@ var _ webhook.Defaulter = &EventIngester{}
 func (r *EventIngester) Default() {
 	eventingesterlog.Info("default", "name", r.Name)
 
-	// TODO(user): fill in your defaulting logic.
+	// image
```

Comment:
Now is probably not the time for it, but a number of these defaulting-logic chunks are so similar, maybe it would be worthwhile to some day make a `SetSpecDefaults(spec *XXXSpec)` - but we'd have to define a generic meta-spec type, or make all those XXXSpec types interfaces?

Reply:
I guess I can do it in the next pass